test/e2e/network_test.go

```diff
@@ -94,13 +61,13 @@ func testEastWestNetworking(t *testing.T) {
 		// test Windows <-> Linux
 		// This will install curl and then curl the windows server.
 		linuxCurlerCommand := []string{"bash", "-c", "yum update; yum install curl -y; curl " + winServerIP}
-		linuxCurlerJob, err := testCtx.createLinuxJob("linux-curler-"+vm.GetCredentials().GetInstanceId(), linuxCurlerCommand)
+		linuxCurlerJob, err := testCtx.createLinuxJob("linux-curler-"+strings.ToLower(node.Status.NodeInfo.MachineID), linuxCurlerCommand)
```

Comment:
Is there a reason why you can't use the node name?

Reply:
Yes the node name contains a "." which is an forbidden character